content/en/docs/how-tos/rbac.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Set Up RBACs on CI Clusters"
+description: How to set up RBACs on CI clusters.
+---
+
+[All the clusters](/docs/getting-started/useful-links/#clusters) for Openshift CI have enabled authentication via Red Hat SSO.
+The [RBAC](https://docs.openshift.com/container-platform/4.6/authentication/using-rbac.html) manifests defined in [the clusters folder of the release
+repo](https://github.com/openshift/release/tree/master/clusters) are applied automatically to the clusters. 
+
+## Rover-groups
+No users are allowed in the RBAC manifests.
```

Comment:
How about:

For privacy reasons, we avoid referring to specific usernames in all RBAC manifests stored in the repository. As an enforced convention, only `Groups` are allowed to be subjects of the RoleBinding and ClusterRoleBinding manifests. 

For the same privacy reasons, we disallow maintaining `Group` manifests directly in the repository. Instead, the users need to use the Red Hat's Rover Groups feature to maintain the list of RH users belonging to a group, and Test Platform tooling will maintain the Group resources on OpenShift CI cluster to contain users corresponding to the Rover group.

Or somethign similar.

Reply:
Updated.